openreview/conference/builder.py

```diff
@@ -2458,13 +2463,31 @@ def get_signatures_regex(self, conference, number):
 
         committee.append(conference.get_anon_reviewer_id(number=number, anon_id='.*'))
 
-        if self.authors:
+        if self.Readers.AUTHORS in self.invitees:
             committee.append(conference.get_authors_id(number))
 
         return '|'.join(committee)
 
     def get_invitees(self, conference, number):
-        return conference.get_committee(number=number, with_authors=self.authors) + [conference.support_user]
+        invitees = [conference.get_program_chairs_id(), conference.support_user]
```

Comment:
the function `get_signatures_regex` must also check the list of invitees to build the regex.